soh/src/code/z_player_lib.c

```diff
@@ -1935,6 +1947,24 @@ void Player_PostLimbDrawGameplay(PlayState* play, s32 limbIndex, Gfx** dList, Ve
                         Player_DrawHookshotReticle(play, this, RETICLE_MAX);
                     }
                 }
+            } else if (CVarGetInteger(CVAR_ENHANCEMENT("BoomerangReticle"), 0) && (this->heldItemAction == PLAYER_IA_BOOMERANG)) {
+                if (Player_HoldsBoomerang(this) != 0) {
+                    MtxF sp44;
+                    s32 pad;
```

Comment:
Neither of these are actually used, so can remove this and the `Matrix_Get(&sp44);` later